projects/plugins/jetpack/class.json-api-endpoints.php

```diff
@@ -1451,8 +1451,11 @@ public function get_author( $author, $show_email_and_ip = false ) {
 				$nice       = $user->user_nicename;
 			}
 			if ( defined( 'IS_WPCOM' ) && IS_WPCOM && ! $is_jetpack ) {
-				$active_blog = get_active_blog_for_user( $id );
-				$site_id     = $active_blog->blog_id;
+				$active_blog = apply_filters( 'wpcom_pre_get_active_blog_for_user', false, $id );
+				if ( false === $active_blog ) {
+					$active_blog = get_active_blog_for_user( $id );
+				}
+				$site_id = $active_blog->blog_id;
```

Comment:
I was looking at `get_active_blog_for_user` code: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qvapyhqrf%2Szf%2Qshapgvbaf.cuc%3Se%3Qrp171832%26zb%3Q927%26sv%3Q45%2348-og and it looks like it can return a `void` value if user has no blogs. Maybe it should go into another PR, but we should definitively check here to see if `$active_blog` is a `WP_Site` object.

Reply:
Added check here b5615314159f98dda12785dbef4efe2dc2d3b055

Reply:
Nice thanks! Dale had another comment on this check so I am closing this as resolved